### PR TITLE
docs(rust): include documentation for CLI commands

### PR DESCRIPTION
**Pull request by adrianbenavides**
We had the documentation in place but we forgot to include it in the commands definitions.

### CODE REVIEW

File: implementations/rust/ockam/ockam_command/src/tcp/connection/list.rs [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
-use crate::CommandGlobalOpts;
+use crate::{docs, CommandGlobalOpts};
 use anyhow::Context;
 use clap::Args;
 use cli_table::{print_stdout, Cell, Style, Table};
 use ockam_api::nodes::models;
 use ockam_api::nodes::models::transport::TransportStatus;
 use ockam_core::api::Request;
 
+const AFTER_LONG_HELP: &str = include_str!("./static/create/after_long_help.txt");
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/tcp/connection/list.rs]: s/create/list

File: implementations/rust/ockam/ockam_command/src/tcp/connection/delete.rs [hunk 2]
```diff
@@ -1,13 +1,15 @@
 use crate::node::{get_node_name, initialize_node_if_default};
 use crate::util::{extract_address_value, node_rpc, Rpc};
-use crate::{node::NodeOpts, CommandGlobalOpts};
+use crate::{docs, node::NodeOpts, CommandGlobalOpts};
 use clap::Args;
 use ockam_api::nodes::models;
 use ockam_core::api::Request;
 
+const AFTER_LONG_HELP: &str = include_str!("./static/create/after_long_help.txt");
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/tcp/connection/delete.rs]: s/create/delete

File: implementations/rust/ockam/ockam_command/src/tcp/connection/show.rs [hunk 3]
```diff
@@ -7,10 +7,13 @@ use ockam_api::nodes::models;
 use ockam_core::api::Request;
 
 use crate::util::{node_rpc, Rpc};
-use crate::CommandGlobalOpts;
+use crate::{docs, CommandGlobalOpts};
+
+const AFTER_LONG_HELP: &str = include_str!("./static/create/after_long_help.txt");
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/tcp/connection/show.rs]: s/create/show

File: implementations/rust/ockam/ockam_command/src/tcp/inlet/delete.rs [hunk 4]
```diff
@@ -1,15 +1,18 @@
...
 use crate::util::{extract_address_value, node_rpc, Rpc};
-use crate::CommandGlobalOpts;
 use crate::Result;
+use crate::{docs, CommandGlobalOpts};
 use clap::Args;
 use colorful::Colorful;
 use ockam::Context;
 use ockam_core::api::{Request, RequestBuilder};
 
+const AFTER_LONG_HELP: &str = include_str!("./static/create/after_long_help.txt");
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/tcp/inlet/delete.rs]: s/create/delete

File: implementations/rust/ockam/ockam_command/src/tcp/inlet/list.rs [hunk 5]
```diff
@@ -1,15 +1,18 @@
...
-use crate::CommandGlobalOpts;
+use crate::{docs, CommandGlobalOpts};
 use anyhow::anyhow;
 use clap::Args;
 use ockam_api::nodes::models;
 use ockam_api::route_to_multiaddr;
 use ockam_core::api::Request;
 use ockam_core::Route;
 
+const AFTER_LONG_HELP: &str = include_str!("./static/create/after_long_help.txt");
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/tcp/inlet/list.rs]: s/create/list

File: implementations/rust/ockam/ockam_command/src/tcp/inlet/show.rs [hunk 6]
```diff
@@ -1,17 +1,19 @@
...
-use crate::CommandGlobalOpts;
 use crate::Result;
+use crate::{docs, CommandGlobalOpts};
 use clap::Args;
 use ockam::{Context, Route};
 use ockam_api::nodes::models::portal::InletStatus;
 use ockam_api::route_to_multiaddr;
 use ockam_core::api::{Request, RequestBuilder};
 
+const AFTER_LONG_HELP: &str = include_str!("./static/create/after_long_help.txt");
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/tcp/inlet/show.rs]: s/create/show

File: implementations/rust/ockam/ockam_command/src/tcp/listener/delete.rs [hunk 7]
```diff
@@ -8,10 +8,13 @@ use ockam_core::api::Request;
 
 use crate::node::{get_node_name, initialize_node_if_default};
 use crate::util::{node_rpc, Rpc};
-use crate::{node::NodeOpts, CommandGlobalOpts};
+use crate::{docs, node::NodeOpts, CommandGlobalOpts};
+
+const AFTER_LONG_HELP: &str = include_str!("./static/create/after_long_help.txt");
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/tcp/listener/delete.rs]: s/create/delete

File: implementations/rust/ockam/ockam_command/src/tcp/listener/list.rs [hunk 8]
```diff
@@ -5,10 +5,13 @@ use ockam_api::nodes::models::transport::{TransportList, TransportStatus};
 
 use crate::node::{get_node_name, initialize_node_if_default, NodeOpts};
 use crate::util::{api, node_rpc, Rpc};
-use crate::CommandGlobalOpts;
+use crate::{docs, CommandGlobalOpts};
+
+const AFTER_LONG_HELP: &str = include_str!("./static/create/after_long_help.txt");
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/tcp/listener/list.rs]: s/create/list

File: implementations/rust/ockam/ockam_command/src/tcp/listener/show.rs [hunk 9]
```diff
@@ -7,10 +7,13 @@ use ockam_api::nodes::models;
 use ockam_core::api::Request;
 
 use crate::util::{node_rpc, Rpc};
-use crate::CommandGlobalOpts;
+use crate::{docs, CommandGlobalOpts};
+
+const AFTER_LONG_HELP: &str = include_str!("./static/create/after_long_help.txt");
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/tcp/listener/show.rs]: s/create/show

File: implementations/rust/ockam/ockam_command/src/tcp/outlet/delete.rs [hunk 10]
```diff
@@ -1,15 +1,18 @@
...
 use crate::util::{extract_address_value, node_rpc, Rpc};
-use crate::CommandGlobalOpts;
 use crate::Result;
+use crate::{docs, CommandGlobalOpts};
 use clap::Args;
 use colorful::Colorful;
 use ockam::Context;
 use ockam_core::api::{Request, RequestBuilder};
 
+const AFTER_LONG_HELP: &str = include_str!("./static/create/after_long_help.txt");
```
Comment (etorreborre) [inline implementations/rust/ockam/ockam_command/src/tcp/outlet/delete.rs]: s/create/delete

**Review by etorreborre — COMMENTED** [review]
I think there are a few places where the link goes to the wrong file

**Comment by adrianbenavides** [comment]
@etorreborre all fixed now!

**Review by etorreborre — APPROVED** [review]
:shipit:

**Pull request merged**